README.md

```diff
@@ -40,6 +40,15 @@
 
   The Language Server for this client can be [found here](https://github.com/angelozerr/lsp4xml).
 
+## Contributing
+
+This is an open source project open to anyone. Contributions are extremely welcome!
+
+For information on getting started, refer to the [CONTRIBUTING instructions](CONTRIBUTING.md).
+
+
+Also, you can contribute your own VSCode extension to enhance the existing features by following the instructions [here](https://github.com/angelozerr/lsp4xml/wiki/Language-Server-Protocol-Extensions).
```

Comment:
nope. for that you'd need to provide support for extension contributions, i.e vscode-xml would need to add external jars  (collected from other extensions) to the server's classpath, when starting the server. Since this is not supported, just remove that paragraph